api/resource/iam_policy.rb

```diff
@@ -54,6 +54,10 @@ class IamPolicy < Api::Object
       # compareSelfLinkOrResourceName
       attr_reader :custom_diff_suppress
 
+      # Some resources (IAP) use fields named differently from the parent resource.
+      # This allows us to customize the test/example attributes of the resource by specifying a file
+      attr_reader :custom_tf_config
```

Comment:
```suggestion
      attr_reader :example_config_body_iam_attributes
```

Reply:
Or `example_config_body`. `custom_tf_config` implies it's a replacement for the whole config, not just a part of it.